src/main/java/org/tron/common/runtime/vm/CallCreate.java

```diff
@@ -25,14 +25,14 @@
 
     final byte[] data;
     final byte[] destination;
-    final byte[] gasLimit;
+    final byte[] energyLimit;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Explicitly declare the visibility for "energyLimit". ([squid:S2039](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2039))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1324&issues=AWVbUev7ZiUEgfxFv6m_&open=AWVbUev7ZiUEgfxFv6m_)

[ISSUE_KEY]: AWVbUev7ZiUEgfxFv6m_